packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -32,9 +39,34 @@ export const getAssetUrl = (path: string) => {
         ? // @ts-ignore
           `${window.Progressive.buildOrigin as string}`
         : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}/`
+
     return path ? `${publicPath}${path}` : publicPath
 }
 
+  
+// TODO: Once we establish that we have a new @salesforce/pwa-kit-extensibility package, we can move this utility to
+// it as to not have direct references to extensibilty in the sdk. This will also reduce duplicate code.
+/**
+ * Get the URL that should be used to load a static asset from the bundle.
+ *
+ * @param {string} path - relative path from the build directory to the asset
+ * @function
+ * @returns {string}
+ */
+export const getStaticAssetUrl = (path: string, opts: GetAssetUrlOptions) => {
```

Comment:
The reason that I created this new utility was because it seemed awkward to use the existing utility `getAssetUrl` and pass in the extension prefix to it in the path like this `getAssetUrl('/static/__extensions/@salesforce/extension-test/asset/path')`.

I didn't like this because `__extensions` is private (we need that so there are no collisions in the assets) and the end developer shouldn't know about it.